docs/_docs/02_testing-best-practices/iterating-locally-using-docker.md

```diff
@@ -19,13 +19,13 @@ Here are some techniques we use with Docker:
 
 - If your script is used in a Packer template, add a [Docker
   builder](https://www.packer.io/docs/builders/docker.html) to the template so you can create a Docker image from the
-  same code. See the [Packer Docker Example]({{ site.baseurl }}/examples/code-examples/packer-docker-example/) for working sample code.
+  same code. See the [Packer Docker Example](https://github.com/gruntwork-io/terratest/tree/master/examples/packer-docker-example) for working sample code.

```

Comment:
Huh... I could've sworn we had a way to direct-link to the [examples on this page](https://terratest.gruntwork.io/examples/), but I guess not!

Reply:
I looked and looked and couldn't find it. Then I just resorted to linking to GH. Better than a broken link. :smile:

Reply:
Agreed!